base/applications/taskmgr/graphctl.c

```diff
@@ -6,676 +6,334 @@
-    this->m_dUpperLimit = 100.0;
-    this->m_dRange      =  this->m_dUpperLimit - this->m_dLowerLimit;  /*  protected member variable */
-
-    /*  m_nShiftPixels determines how much the plot shifts (in terms of pixels)  */
-    /*  with the addition of a new data point */
-    this->m_nShiftPixels     = 4;
-    this->m_nHalfShiftPixels = this->m_nShiftPixels / 2;  /*  protected */
-    this->m_nPlotShiftPixels = this->m_nShiftPixels + this->m_nHalfShiftPixels;  /*  protected */
-
-    /*  Set how big the grid boxes are  */
-    this->m_nGridXPixels = 12;
-    this->m_nGridYPixels = 12;
-
-    /*  Task manager should scroll grid with the graph. This variable contains the  */
-    /*  amount of pixels the grid paint origin should shift to left  */
-    this->m_nGridOffsetPixels = 0;
-
-    /*  background, grid and data colors */
-    /*  these are public variables and can be set directly */
-    this->m_crBackColor = RGB(  0,   0,   0);  /*  see also SetBackgroundColor */
-    this->m_crGridColor = RGB(  0, 128,  64);  /*  see also SetGridColor */
-    this->m_crPlotColor[0] = RGB(255, 255, 255);  /*  see also SetPlotColor */
-    this->m_crPlotColor[1] = RGB(100, 255, 255);  /*  see also SetPlotColor */
-    this->m_crPlotColor[2] = RGB(255, 100, 255);  /*  see also SetPlotColor */
-    this->m_crPlotColor[3] = RGB(255, 255, 100);  /*  see also SetPlotColor */
-
-    /*  protected variables */
-    for (i = 0; i < MAX_PLOTS; i++)
-    {
-        this->m_penPlot[i] = CreatePen(PS_SOLID, 0, this->m_crPlotColor[i]);
-    }
-    this->m_brushBack = CreateSolidBrush(this->m_crBackColor);
-
-    /*  public member variables, can be set directly  */
-    strcpy(this->m_strXUnitsString, "Samples");  /*  can also be set with SetXUnits */
-    strcpy(this->m_strYUnitsString, "Y units");  /*  can also be set with SetYUnits */
-
-    /*  protected bitmaps to restore the memory DC's */
-    this->m_bitmapOldGrid = NULL;
-    this->m_bitmapOldPlot = NULL;
-}
-
-void GraphCtrl_Dispose(TGraphCtrl* this)
-{
-    int plot;
-
-    for (plot = 0; plot < MAX_PLOTS; plot++)
-        DeleteObject(this->m_penPlot[plot]);
-
-    /*  just to be picky restore the bitmaps for the two memory dc's */
-    /*  (these dc's are being destroyed so there shouldn't be any leaks) */
-
-    if (this->m_bitmapOldGrid != NULL) SelectObject(this->m_dcGrid, this->m_bitmapOldGrid);
-    if (this->m_bitmapOldPlot != NULL) SelectObject(this->m_dcPlot, this->m_bitmapOldPlot);
-    if (this->m_bitmapGrid    != NULL) DeleteObject(this->m_bitmapGrid);
-    if (this->m_bitmapPlot    != NULL) DeleteObject(this->m_bitmapPlot);
-    if (this->m_dcGrid        != NULL) DeleteDC(this->m_dcGrid);
-    if (this->m_dcPlot        != NULL) DeleteDC(this->m_dcPlot);
-    if (this->m_brushBack     != NULL) DeleteObject(this->m_brushBack);
+void TmgcCreate(PTM_GRAPH_CONTROL inst, HWND hWnd, HWND hParentWnd, PTM_FORMAT fmt){
+	HDC	hdc, hdcg;
+	HBITMAP	hbmOld;
+	PBYTE	t;
+	UINT	Size;
+	INT	p;
+	RECT	rc;
```

Comment:
Please fix coding style according to this: https://reactos.org/wiki/Coding_Style

(also, indent with 4 spaces instead of tabs)